lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -838,20 +838,21 @@ def _mean_and_variance_to_probabilities(mean_values, variance_values,
             variance_values (iris.cube.Cube):
                 Variance for the calibrated forecast.
             probability_cube_template (iris.cube.Cube):
-                A probability cube that has the threshold coordinate, and
-                attribute relative_to_threshold, that match the desired output
-                cube format.
+                A probability cube that has a threshold coordinate with an
+                spp__relative_to_threshold attribute that matches the desired
+                output cube format.
```

Comment:
Could I request a tiny expansion of the comment. I know what it means, but the uninitiated might not understand this comment in isolation; e.g.
```A probability cube that has a threshold coordinate, where the probabilities are defined as above or below the threshold by the spp__relative_to_threshold attribute. This cube matches the desired output cube format.```

Reply:
I've updated the docstring as suggested.